app/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,57 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="custom_switch_text" parent="@android:style/TextAppearance.Small">
+        <item name="android:textColor">#FFFFFF</item>
+    </style>
+
+    <style name="textBig" parent="@style/TextAppearance.AppCompat.Large">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:textColor">@android:color/black</item>
+    </style>
+    <style name="TextMedium" parent="@style/TextAppearance.AppCompat.Subhead">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:textColor">@android:color/black</item>
+    </style>
+    <style name="TextSummary" parent="@style/TextAppearance.AppCompat.Caption">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginLeft">@dimen/activity_horizontal_margin</item>
+        <item name="android:layout_marginStart">@dimen/activity_horizontal_margin</item>
+        <item name="android:typeface">monospace</item>
+        <item name="android:textSize">20sp</item>
+    </style>
+    <style name="TextSummaryLabel" parent="@style/TextSummary">
+        <item name="android:textColor">@color/colorLabelGreyDark</item>
+        <item name="android:layout_marginTop">@dimen/activity_vertical_margin</item>
+
+    </style>
+    <style name="TextSummaryValue" parent="@style/TextSummary">
+        <item name="android:textColor">@android:color/black</item>
+
+    </style>
+    <style name="custom_switch">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginLeft">@dimen/activity_vertical_margin</item>
+        <item name="android:layout_marginStart">@dimen/activity_vertical_margin</item>
+        <item name="android:showText">true</item>
+        <item name="android:switchTextAppearance">@style/custom_switch_text</item>
+        <item name="android:textOff">No</item>
+        <item name="android:textOn">Yes</item>
+        <item name="android:thumb">@drawable/custom_switch</item>
+        <item name="android:track">@drawable/custom_track</item>
+    </style>
+
+
```

Comment:
Either keep all the XML style elements together or have a single line between each. Let's keep the code style consistent